src/iidm/ThreeWindingsTransformerLeg.cpp

```diff
@@ -202,6 +237,7 @@ void Leg::setRatioTapChanger(std::unique_ptr<RatioTapChanger>&& ratioTapChanger)
 
 Leg& Leg::setTransformer(ThreeWindingsTransformer& transformer) {
     m_transformer = transformer;
+    m_operationalLimitsHolder = stdcxx::make_unique<OperationalLimitsHolder>(m_transformer.get(), stdcxx::format("limits%1%", m_legNumber));
```

Comment:
This should explain why you need a std::unique_ptr. To be check if we can refactor the code to avoid it, using a `stdcxx::reference` for instance.